### PR TITLE
Fix missing ability to get pilot out of mecha manually as MMI 

### PR DESCRIPTION
**Pull request by ErdGinalD**
## What Does This PR Do
Fixing not being able to get pilot out of mecha using tools, as Wiki said to, but i remove this line because this not in the game. After fixing i return old description
image

## Why It's Good For The Game
Wiki said we can get out pilots and MMI from mecha, so we do it. Also this is clearly missing part of interaction.

## Images of changes
image

## Testing
copypasta from downstream, also tested and it work. Now you can pry out miner from mecha and stole his mech(if you agile enough)

## Changelog
:cl:
fix: Fix missing ability to pull out pilot from mech as same as MMI.
/:cl:

### CODE REVIEW

**Comment by Sirryan2002** [comment]
failed test
testing

enough said :/

**Comment by ErdGinalD** [comment]
> failed test testing
> 
> enough said :/
> 
> I recommend filling out the template properly, documenting all changes (you are adding new functionality) in the description, and then testing on our version of the codebase + documenting it.

old one, this is more fix, more explanations when im done with it

**Review by Sirryan2002 — APPROVED** [review]
code looks good. Did you test to make sure you can still pry out MMIs?

**Comment by ErdGinalD** [comment]
> code looks good. Did you test to make sure you can still pry out MMIs?

yes, all worked well. At least in local with 2 users. If human inside, it pry out with words "driver", if MMI/posy - MMI

File: code/game/mecha/mecha.dm [hunk 1]
```diff
@@ -831,11 +831,17 @@
 	else if(state == MECHA_OPEN_HATCH)
 		state = MECHA_BOLTS_UP
 		to_chat(user, "You close the hatch to the power unit")
+	else if(ishuman(occupant))
+		user.visible_message("<span class='notice'>[user] begins levering out the driver from the [src].</span>", "<span class='notice'>You begin to lever out the driver from the [src].</span>")
+		to_chat(occupant, "<span class='warning'>[user] is prying you out of the exosuit!</span>")
+		if(I.use_tool(src, user, 8 SECONDS, volume = I.tool_volume))
+			user.visible_message("<span class='notice'>[user] pries the driver out of the [src]!", "<span class='notice'>You finish removing the driver from the [src]!</span>")
```
Comment (Burzah) [inline code/game/mecha/mecha.dm:838]:
```suggestion
			user.visible_message("<span class='notice'>[user] pries the driver out of the [src]!</span>", "<span class='notice'>You finish removing the driver from the [src]!</span>")
```

**Review by FunnyMan3595 — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Pull request merged**